src/Microcosm.js

```diff
@@ -134,8 +134,8 @@ Microcosm.prototype = {
    * Clear all outstanding transactions and assign base state
    * to a given object (or getInitialState())
    */
-  reset(state=this.getInitialState(), transactions=[]) {
-    this.transactions = transactions
+  reset(state=this.getInitialState()) {
+    this.transactions = []
```

Comment:
Not related to this PR, but I went a head and removed the default transaction list argument from `reset`. Eventually, I'd like to be able to inject history when you reset so that you can rehydrate an app with "undo" state.